docs/ROADMAP.md

```diff
@@ -0,0 +1,442 @@
+PRESC Project Roadmap
+=====================
+
+This is an overview of the features planned for integration into PRESC.
+It is intended to give a high-level description of how these will work and
+sample use cases.
+Prioritization and implementation details are maintained in the repo
+[issues](https://github.com/mozilla/PRESC/issues).
+
+At the core of PRESC is a collection of analyses that can be run on a given
+statistical model and dataset to inform the developer on different aspects of
+the model's performance and behaviour.
+The two main intended uses are a graphical presentation in a report and a
+pass/fail determination obtained by comparing the results against threshold
+values determined by the user.
+In either case, it would be up to the user to decide on a course of action to
+correct deficiencies in the model surfaced by these evaluations.
+
+Planned analyses are described below, grouped by theme.
+Some of these will lend themselves to multiple possible visualizations or
+summaries, while others will be applicable in a single clear way.
+As of right now, the details are fleshed out to varying extents.
```

Comment:
remove this line:
"As of right now, the details are fleshed out to varying extents."